code/game/objects/items/weapons/secdocs.dm

```diff
@@ -77,3 +77,15 @@
 	if(istype(owner))
 		to_chat(owner, SPAN_NOTICE("You have valuable scientific data on your person. Do not let it fall into the wrong hands."))
 
+/obj/item/magiccatpaperwork
+	name = "Server Access Files"
+	desc = "A power of red tape turn blue, these files give the colonist best access needed to run the stage by themselfs. Like it was added loop hole for one person... Back door access to something only the HC can gain."
+	icon_state = "siezed"
+	icon = 'icons/obj/oddities.dmi'
+	price_tag = 1984
+
+/obj/item/magiccatpaperwork/attack(mob/M as mob, mob/user as mob)
```

Comment:
```suggestion
/obj/item/magiccatpaperwork/attack_self(mob/user as mob)
```

Reply:
Otherwise, you need to smack someone else for it to work.

Reply:
Intented